### DISCUSSION TITLE
Nickelodeon East US

### DISCUSSION THREAD

**Discussion by SkylerDaKat** [iptv-org/community#927] [Q&A]
### Channel ID

NickelodeonEast.us

### Notes (optional)

This will be my last request, so sorry about doing so many qwp
I personally want just the east channel, but if it also includes west (Nickelodeonwest.us) and south (Nickelodeonsouth.us), that would be awesome.

And I'm opening these to see if they are going to be added or give me a reason why it isn't going to be.
Anyways have a great day

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/us_moveonjoy.m3u#L60-L61

https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/us.m3u#L583-L584
